synchro/__init__.py

```diff
@@ -468,6 +468,17 @@ def __init__(self, client, name, **kwargs):
     def client(self):
         return self._client
 
+    def cursor_command(self, *args, **kwargs):
+        if "session" in kwargs:
+            # Workaround for validation added in PYTHON-3228.
+            kwargs["session"] = kwargs["session"].delegate
+        cursor = self.synchronize(self.delegate.cursor_command)(*args, **kwargs)
+        cmd_cursor = CommandCursor(cursor)
+        # Send the initial command as PyMongo expects.
```

Comment:
I believe this part should be removed IIUC. The PyMongo cursor_command method never runs getMore so neither should synchro.

Reply:
There is a `test.test_run_command.TestUnifiedRunCursorCommand.test_accepts_an_explicit_session_that_is_reused_across_getMores` test.

Reply:
That unified test operation works differently because it iterates the resulting cursor returned by run_command, see [here](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#runcursorcommand):
> When executing the provided command, the test runner MUST fully iterate the cursor.

So the getMore part here still needs to be removed.

Reply:
Done